$NetBSD: patch-roff.7,v 1.7 2015/07/19 19:19:10 joerg Exp $

Install roff with 'mandoc-' prefix to avoid conflict with groff.

--- roff.7.orig	2015-03-13 12:38:38.000000000 +0000
+++ roff.7
@@ -19,4 +19,4 @@
-.Dt ROFF 7
+.Dt MANDOC-ROFF 7
 .Os
 .Sh NAME
-.Nm roff
+.Nm mandoc-roff
